modules/RTC/ortc/errors.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Create a class inheriting from Error.
+ */
+function createErrorClass(name) {
+    const klass = class extends Error {
+        /**
+         */
+        constructor(message) {
+            super(message);
+
+            // Override `name` property value and make it non enumerable.
+            Object.defineProperty(this, 'name', { value: name });
+        }
+    };
+
+    return klass;
+}
+
+module.exports = {
```

Comment:
ES6 import/export does not allow exporting objects. Let me figure out how to export individuals. Not hard AFAIK. Will do.